filament/backend/src/vulkan/platform/VulkanPlatform.cpp

```diff
@@ -524,6 +524,28 @@ VkFormatList findAttachmentDepthFormats(VkPhysicalDevice device) {
     return ret;
 }
 
+VkFormatList findBlittableDepthStencilFormats(VkPhysicalDevice device) {
+    std::vector<VkFormat> selectedFormats;
+    VkFormatFeatureFlags const required = VK_FORMAT_FEATURE_DEPTH_STENCIL_ATTACHMENT_BIT |
+            VK_FORMAT_FEATURE_BLIT_SRC_BIT | VK_FORMAT_FEATURE_BLIT_DST_BIT;
+    for (VkFormat format = (VkFormat) 1;;) {
+        if (isVkDepthFormat(format)) {
+            VkFormatProperties props;
+            vkGetPhysicalDeviceFormatProperties(device, format, &props);
+            if ((props.optimalTilingFeatures & required) == required) {
+                selectedFormats.push_back(format);
+            }
+        }
+        format = (VkFormat) (1 + (int) format);
+        if (format == VK_FORMAT_ASTC_12x12_SRGB_BLOCK) {
```

Comment:
Maybe comment that this is the last format provided by VK_VERSION_1_0? And do we want to exclude `VK_FORMAT_ASTC_12x12_SRGB_BLOCK` from this check (maybe this if needs to be before line 539)?

Reply:
yeah, this could definitely be cleaner.  I'll put up a follow-up to clean this up.